entrypoint.sh

```diff
@@ -150,6 +147,7 @@ if [[ "${SSL_MODE}" == "true" ]]; then
     --elasticsearch.ssl.key=/home/elastic/elasticsearch-${EK_VERSION}/config/certs/instance/instance.key \
     --elasticsearch.ssl.verificationMode=certificate \
     --elasticsearch.serviceAccountToken=${KIBANA_SERVICE_TOKEN}
+  # Start Kibana in SSL Mode
```

Comment:
I don't think this comment should be placed here? (why moved it from L138 to L150)

Reply:
The `while()` loop is placed in unison with a ` ... &` stringed command and the comment for POXIS syntax apparently does not run the `nohup` command directly after a comment.

This apparently fixed that issue?